Sources/SpeziLLMLocalDownload/LLMLocalDownloadManager.swift

```diff
@@ -120,12 +127,14 @@ public final class LLMLocalDownloadManager: NSObject {
         }
 
         let repo = Hub.Repo(id: model.hubID)
-        let modelFiles = ["*.safetensors", "config.json"]
-        
-        try await HubApi.shared.snapshot(from: repo, matching: modelFiles) { progress in
-            Task { @MainActor [mutate] in
-                mutate(progress)
+        let modelFiles = ["*.safetensors", "*.json"]
+
+        // important to match the same destination directory as in the `ModelFactory`
+        try await LLMLocalSession.hubApi
+            .snapshot(from: repo, matching: modelFiles) { progress in
+                Task { @MainActor [mutate] in
+                    mutate(progress)
```

Comment:
Open TODOs:
- [ ] Somehow the process is not communicated; stays at 0% until the download is done.
- [ ] Downloading text is also a bit misaligned; small UI fix there.